src/actions.py

```diff
@@ -153,3 +165,38 @@ def delete_selected_action(_action, _target, self):
     names = ", ".join(item.get_name() for item in items)
     self.get_data().delete_items(items)
     self.get_window().add_toast_string(_("Deleted {}").format(names))
+
+
+@Gtk.Template(resource_path="/se/sjoerd/Graphs/ui/smoothen_settings.ui")
```

Comment:
This can probably moved to vala in a separate pr

Reply:
I moved most of it to Vala, but couldn't get the reset button to work. So that's still in the python code for now.